database/migrations/2017_12_04_113343_create_registrations_table.php

```diff
@@ -21,6 +21,7 @@ public function up()
             $table->dateTime('consented_on')->nullable();
             $table->dateTime('fm_chart_on')->nullable();
             $table->dateTime('fm_diary_on')->nullable();
+            $table->dateTime('fm_privacy_on')->nullable();
```

Comment:
This needs to be in a migration to add the field now - we're live, so we can't rerfresh the DB, we have to modify it.